src/test/java/me/karakelley/tictactoe/BoardStateTest.java

```diff
@@ -65,4 +68,10 @@ public void testCombinations() {
     boardState.placeMove("2", "X");
     assertArrayEquals(expectedOutput, boardState.winningCombinations());
   }
+
+  @Test
+  public void testAvailableCells() {
+    ArrayList<Integer> expectedOutput = new ArrayList<>(Arrays.asList(0, 1, 2, 3, 4, 5, 6, 7, 8));
```

Comment:
1. You shouldn't need to wrap this in an `ArrayList` if you just type it as `List<Integer>`.
2. I think there are some missing tests here. This isn't testing the `filter`. Make sure you're only writing the *minimum* amount of code necessary to make your unit tests pass.